experimental/segmenter/src/line_breaker.rs

```diff
@@ -64,6 +65,92 @@ pub enum WordBreakRule {
     KeepAll,
 }
 
+/// Options to tailor line breaking behavior, such as for CSS.
+#[non_exhaustive]
+#[derive(Clone, PartialEq, Eq)]
+pub struct LineBreakOptions {
```

Comment:
It's interesting to see you add a similar struct as I did for gecko's segmenter scaffolding API :)
https://searchfox.org/mozilla-central/rev/21a9b72545da06681db97c4b3a2a6be761f4aae5/intl/lwbrk/Segmenter.h#93-98

Question: I'm curious the reason for this to be `#[non_exhaustive]`. Are you planning to add more fields in your later PR?

Reply:
Because the style guide dictates that options bags should be non_exhaustive, not because we need it right now but because we don't want to be forced to bump the major version number in the future if we add a new field.

https://github.com/unicode-org/icu4x/blob/main/docs/process/style_guide.md#implement-default-and-non_exhaustive--required

Reply:
Thanks for the answer. I completely forget about this section.